src/index.js

```diff
@@ -61,12 +61,12 @@ function App() {
           <SecureRoute
             exact
             path="/myprofile/inventory"
-            component={CurrentInventory}
+            component={Inventory}
           />
           <SecureRoute
             exact
             path="/myprofile/inventory/additem"
```

Comment:
could you please change the path name to /stores/:store_id/inventory ? - also use the store_id via router to fetch the store.